catalog/core/catalog-core-resourcesizeplugin/pom.xml

```diff
@@ -78,7 +78,8 @@
                         </Embed-Dependency>
                         <Private-Package>
                             ddf.catalog.cache.impl,
-                            ddf.catalog.plugin.resourcesize.metacard
+                            ddf.catalog.plugin.resourcesize.metacard,
+                            org.codice.ddf.platform.util
```

Comment:
Any reason we need to private-package `org.codice.ddf.platform.util` here instead of doing like we did elsewhere and embed `platform-util`?